docs/whats-new/dotnet-docs-mod2.md

```diff
@@ -1,95 +1,114 @@
 ---
-title: ".NET docs: What's new for August 2022"
-description: "What's new in the .NET docs for August 2022."
-ms.custom: August-2022
-ms.date: 09/01/2022
+title: ".NET docs: What's new for November 2022"
+description: "What's new in the .NET docs for November 2022."
+ms.custom: November-2022
+ms.date: 12/01/2022
 ---
 
-# .NET docs: What's new for August 2022
+# .NET docs: What's new for November 2022
 
-Welcome to what's new in the .NET docs for August 2022. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the .NET docs for November 2022. This article lists some of the major changes to docs during this period.
 
 ## .NET breaking changes
 
 ### New articles
 
-- [BinaryFormatter serialization APIs produce compiler errors](../core/compatibility/core-libraries/7.0/binaryformatter-apis-produce-errors.md)
-- [All assemblies trimmed by default](../core/compatibility/deployment/7.0/trim-all-assemblies.md)
-
-## .NET Framework
-
-### New articles
-
-- [Install .NET Framework on Windows Server 2022](../framework/install/on-server-2022.md)
-- [Retargeting changes for migration from .NET Framework 4.8 to 4.8.1](../framework/migration-guide/retargeting/4.8-4.8.1.md)
-- [Runtime changes for migration from .NET Framework 4.8 to 4.8.1](../framework/migration-guide/runtime/4.8-4.8.1.md)
+- [BrotliStream no longer allows undefined CompressionLevel values](../core/compatibility/core-libraries/7.0/brotlistream-ctor.md)
+- [Flow direction helper methods removed](../core/compatibility/maui/7.0/flow-direction-apis-removed.md)
+- [Some Windows APIs are removed](../core/compatibility/maui/7.0/iwindowstatemanager-apis-removed.md)
+- [Constructors accept base interface instead of concrete type](../core/compatibility/maui/7.0/mauiwebviewnavigationdelegate-constructor.md)
+- [ScrollToRequest property renamed](../core/compatibility/maui/7.0/scrolltorequest-property-rename.md)
+- [New UpdateBackground parameter](../core/compatibility/maui/7.0/updatebackground-parameter.md)
+- [System.diagnostics entry in app.config](../core/compatibility/configuration/7.0/diagnostics-config-section.md)
+- [API obsoletions with default diagnostic ID (.NET 7)](../core/compatibility/core-libraries/7.0/obsolete-apis-with-default-diagnostic.md)
+- [System.Runtime.CompilerServices.Unsafe NuGet package](../core/compatibility/core-libraries/7.0/unsafe-package.md)
+- [Environment variable prefixes](../core/compatibility/extensions/7.0/environment-variable-prefix.md)
+- [Legacy FileStream strategy removed](../core/compatibility/core-libraries/7.0/filestream-compat-switch.md)
+- [Time fields on symbolic links](../core/compatibility/core-libraries/7.0/symbolic-link-timestamps.md)
 
 ## .NET fundamentals
 
 ### New articles
```

Comment:
How come What's new in .NET 7 didn't get picked up? https://github.com/dotnet/docs/blob/main/docs/core/whats-new/dotnet-7.md

Reply:
There's a filter in the logic so it doesn't pick up anything in the *what's new* folder. (Otherwise, it always picks up the most recent article in this series. Which felt bad. Let's add it by hand.

Reply:
@gewarren I just added a link to the what's new in docs for .NET 7. I set this PR to auto-merge.

If you approve it, it will ship

Reply:
> There's a filter in the logic so it doesn't pick up anything in the _what's new_ folder. (Otherwise, it always picks up the most recent article in this series. Which felt bad. Let's add it by hand.

I was referring to the "what's new in the product" page, not "what's new in docs", which I think is important to be picked up in these docs. That's in a different "whats-new" folder - can we make the exclusion more specific so we don't have to add the "what's new in the product" file manually in the future?

Reply:
Gotcha.  I made a note of that.